### PR TITLE
Test libraries

### PR DESCRIPTION
**Pull request by vinicios-gomes**
- Configured Cypress library for integration tests

### CODE REVIEW

File: tests/e2e/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// if you need a custom webpack configuration you can uncomment the following import
+// and then use the `file:preprocessor` event
+// as explained in the cypress docs
+// https://docs.cypress.io/api/plugins/preprocessors-api.html#Examples
+
+// /* eslint-disable import/no-extraneous-dependencies, global-require */
+// const webpack = require('@cypress/webpack-preprocessor')
+
+module.exports = (on, config) => {
+	// on('file:preprocessor', webpack({
```
Comment (vitormattos) [inline tests/e2e/plugins/index.js:13]: Comment in unused code

File: tests/e2e/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (vitormattos) [inline tests/e2e/support/commands.js:1]: Is correct maintain all lines commented

File: jest.config.js [hunk 3]
```diff
@@ -0,0 +1,194 @@
+/*
```
Comment (vitormattos) [inline jest.config.js:1]: Is correct maintain much lines commented

File: tests/e2e/plugins/index.js (hunk 1 above)
Comment (vinicios-gomes) [reply tests/e2e/plugins/index.js:13]: Done.

File: tests/e2e/support/commands.js (hunk 2 above)
Comment (vinicios-gomes) [reply tests/e2e/support/commands.js:1]: Done.

**Pull request merged**